### DISCUSSION TITLE
Using @sap/ux-ui5-fe-mockserver-middleware with odata v2 model cards

### DISCUSSION THREAD

**Discussion by UBSusmitha** [SAP/open-ux-odata#670] [Q&A]
Odata v2 model cards are not being rendered in Jenkins after I started using @sap/ux-ui5-fe-mockserver-middleware. 
OPA test cases are failing only in jenkins, while it works fine in local. 

Any support/help would be appreciated!

**Comment by nlunets** [comment]
Have you checked https://github.com/SAP/open-ux-odata/tree/main/packages/ui5-middleware-fe-mockserver#special-case-within-karma ?

**Reply by UBSusmitha** [reply]
Thanks for the reply. I have tried to change beforeMiddleware: csp instead of afterMiddleware: compression in ui5.yaml file. After this change also in jenkins OPAs fail.
